src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -204,6 +204,11 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   const onBarCodeScanned: BarCodeScannedCallback = (event) => {
     if (isFocused && isScanningEnabled && event.data) {
       onCheck(event.data);
```

Comment:
Shouldn't this be removed? In the case where the type is not `Code39`, we won't be able to perform `onCheck` with the `passportId`, if this is still present.

```suggestion
```

Reply:
oh shoot. thankss